7-gce/config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This needs to be pushed to the GCE instance. Should we tell users to add it themselves instead?